### PR TITLE
Clarify subplot syntax vs Quadmesh colorbar

### LINKED ISSUE #275: Issue on page /core/matplotlib/annotations-colorbars-layouts.html
In the "Shared colorbars" section, there is this note:
https://github.com/ProjectPythia/pythia-foundations/blob/9c3a75be8dffce205c714674942d7ec9323bdd92/core/matplotlib/annotations-colorbars-layouts.ipynb?short_path=ac96277#L418-L429

I think this is slightly misleading, as `hist1[3]` is not "subplot syntax"', as `hist1` is a tuple returned by `hist2d` and `hist1[3]` is a ` matplotlib.collections.QuadMesh` (cf https://matplotlib.org/stable/api/_as_gen/matplotlib.axes.Axes.hist2d.html).

This was a little confusing to me when I read it.

### PR DESCRIPTION
**Pull request by jukent**
Thanks for pointing out the inaccuracy @phil-blain Does this fix clarify things?

Closes #275

### CODE REVIEW

File: core/matplotlib/annotations-colorbars-layouts.ipynb [hunk 1]
```diff
@@ -426,7 +426,9 @@
    "id": "1662bf3c",
```
Comment (mgrover1) [inline core/matplotlib/annotations-colorbars-layouts.ipynb:426]: <p>Can you make "Documentation" lowercase?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ProjectPythia/pythia-foundations/pull/282/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='31'/>

**Review by mgrover1 — CHANGES_REQUESTED** [review]
One small suggestion - good catch!

File: core/matplotlib/annotations-colorbars-layouts.ipynb (hunk 1 above)
Comment (jukent) [reply core/matplotlib/annotations-colorbars-layouts.ipynb:426]: Done!

**Review by mgrover1 — APPROVED** [review]
Thanks @jukent for making that change!

**Review by dcamron — APPROVED** [review]
👍

**Pull request merged**